tests/java-tests/src/test/java/test/ClientTest.java

```diff
@@ -41,8 +67,153 @@ public void before() {
   @Test
   public void simple() {
     mockServer.enqueue(new MockResponse("{\"data\": {\"random\": 42}}"));
+    ApolloResponse<GetRandomQuery.Data> queryResponse = Rx2Apollo.rxSingle(
```

Comment:
Proposal to rename `rxSingle` -> `single` in Java (not Kotlin) as it's a mouthful at the moment.

Reply:
Good call! 👍